### DISCUSSION TITLE
Wpkg bulk deployment script? [Question]

### DISCUSSION THREAD

**Discussion by NenadV84** [2fast-team/2fast#99] [General]
Hi, we are having issues downloading app from MS store.
Can you send us some instruction for maybe wpkg installation, so we can deploy bulk installation on multiple PC’s?
If not, please how to install manually from zip file?

Thanks,
Nenad

**Comment by jp-weber** [comment]
Which variant was tried to do the download through the MS Store?

**Reply by NenadV84** [reply]
Hi. Our company policy forbides MS store apps download. We preffer wpkg installation deployment on PC's.

**Reply by NenadV84** [reply]
I tried to install manually from zip file but power shell install file did not worked for us. Can you post some instractions maybe?

**Reply by jp-weber** [reply]
Are only the clients restricted for the access of the MS Store? I would suggest to [download the app via winget CLI](https://learn.microsoft.com/en-us/windows/package-manager/winget/), which have the MS Store as source.
```
winget install 9P9D81GLH89Q
```
Alternative you can the app installed via the installer files. First the certificate must be installed, see:
https://github.com/2fast-team/2fast/wiki/How-to-install-.msix-package

I have never worked with wpkg before, so unfortunately I can't offer any instructions.

**Comment by NenadV84** [comment]
MS store source is forbiden.
So, after certificate installation we need to run only install.ps file?

**Reply by jp-weber** [reply]
If you execute this via the powershell script, the certificate would already be installed (assuming admin rights).
